### PR TITLE
feat: loading is added in the button

### PR DESCRIPTION
**Pull request by palashgdev**
Close https://github.com/SigNoz/signoz/issues/1914

### CODE REVIEW

File: frontend/src/container/LogControls/index.tsx [hunk 1]
```diff
@@ -56,24 +58,43 @@ function LogControls(): JSX.Element | null {
...
+				size="small"
+				type="link"
+				onClick={handleNavigateNext}
+			>
 				Next <RightOutlined />
 			</Button>
 			<Select
-				style={{ width: 120 }}
+				loading={isLoading || isLoadingAggregate}
+				disabled={isLoading || isLoadingAggregate}
```
Comment (jyash97) [inline frontend/src/container/LogControls/index.tsx:91]: When loading is true the button would be disabled, maybe we can remove this

Comment (palashgdev) [reply frontend/src/container/LogControls/index.tsx]: yup this one is updated

File: frontend/src/container/LogControls/config.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const ITEMS_PER_PAGE_OPTIONS = [25, 50, 100, 200];
```
Comment (jyash97) [inline frontend/src/container/LogControls/config.ts:1]: This could be placed in the same component itself, unless we will use this elsewhere

**Review by jyash97 — APPROVED** [review]

File: frontend/src/container/LogControls/config.ts (hunk 2 above)
Comment (palashgdev) [reply frontend/src/container/LogControls/config.ts:1]: i guess this is more relevant to the component itself... but don't want to put all the stuff in one place

**Comment by pranay01** [comment]
LGTM functionality wise, though it seemed to me that the loading time on new logs API call is a bit slow. 

@nityanandagohain Can you also test this. If the loading  time on API calls is higher?

**Review by pranay01 — DISMISSED** [review]

**Comment by pranay01** [comment]
I tested the performance in dev environment along with Palash. The performance seemed to be in the same ballpark as current `develop` branch.

**Comment by nityanandagohain** [comment]
Found a bug, not sure if it is related to this @palashgdev 
* Click on next
* click on previous -> previous -> previous ( you will reach a state were it will say no logs found)
* The reason for this is wrong timestamps are getting sent in the API request.

image

**Comment by palashgdev** [comment]
No @nityanandagohain this is not related to this PR

**Comment by nityanandagohain** [comment]
@pranay01 the time is fine for filter -> result, but I feel the prev/next buttons experience is not good, but we can address that in different issue.

**Comment by pranay01** [comment]
> @pranay01 the time is fine for filter -> result, but I feel the prev/next buttons experience is not good, but we can address that in different issue.

Ok. Yeah, that should be a different issue. Can you create one for this @nityanandagohain

**Review by pranay01 — APPROVED** [review]

**Pull request merged**